dev/benchmarks/macrobenchmarks/lib/web_benchmarks.dart

```diff
@@ -50,7 +53,36 @@ final Map<String, RecorderFactory> benchmarks = <String, RecorderFactory>{
     BenchTextCachedLayout.canvasBenchmarkName: () => BenchTextCachedLayout(useCanvas: true),
     BenchBuildColorsGrid.domBenchmarkName: () => BenchBuildColorsGrid.dom(),
     BenchBuildColorsGrid.canvasBenchmarkName: () => BenchBuildColorsGrid.canvas(),
-  }
+  },
+
+  // The following benchmarks are for the Flutter Gallery.
+  'gallery_studies_perf': () => GalleryRecorder(
```

Comment:
Wondering if for consistency, this should use "v2" or "new" so as to not be confused with other benchmarks

Reply:
Yes! I'll use `flutter_v2`. Thank you!

Reply:
Done.